tests/functional/client.js

```diff
@@ -60,4 +60,33 @@ describe('Bucket Client tests', function testClient() {
             return done(new Error('Did not fail as expected'));
         });
     });
+
+    it('should get logs from bucketd', done => {
```

Comment:
Yes, you're right, it does not run in ci. A functional test of Metadata will use the 'getRaftLogs` function: https://github.com/scality/MetaData/pull/1153